### PR TITLE
Module level import not at top of file

### PR DESCRIPTION
**Pull request by rosswhitfield**
This fixes all flake8-E402 warnings about imports not being at the top of the file.

**To test:**
- Code review
# **_Does not need to be in the release notes.**_

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Calibration/Examples/TubeCalibDemoWish1.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # We base the ideal tube on one tube of this door.
 #
 import tube
-reload(tube)
+reload(tube) # noqa
```
Comment (AndreiSavici) [inline scripts/Calibration/Examples/TubeCalibDemoWish1.py]: Do you even need `reload`?

File: Framework/PythonInterface/plugins/algorithms/RefLReduction.py [hunk 2]
```diff
@@ -7,10 +7,10 @@
 import sys
 import os
 sys.path.insert(0,os.path.dirname(__file__))
-import sfCalculator
+import sfCalculator # noqa
 sys.path.pop(0)
 
-from mantid.kernel import *
+from mantid.kernel import * # noqa
```
Comment (AndreiSavici) [inline Framework/PythonInterface/plugins/algorithms/RefLReduction.py]: You can move this line before, after line 4

**Comment by AndreiSavici** [comment]
:shipit:

**Comment by KarlPalmen** [comment]
It seems that ` # noqa` means ``shut up Flake8.``

**Comment by rosswhitfield** [comment]
@KarlPalmen Yes According to [this page](https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes) it should only work for warnings marked with **(^)** but it appears to work for any warning.

    (^) These checks can be disabled at the line level using the # noqa special comment. This possibility should be reserved for special cases.

**Pull request merged**